$OpenBSD: patch-libavformat_oggdec_h,v 1.1 2011/08/28 11:17:48 sthen Exp $

Make sure start time correction is applied once to each stream.

--- libavformat/oggdec.h.orig	Wed Aug 24 18:15:40 2011
+++ libavformat/oggdec.h	Wed Aug 24 18:16:01 2011
@@ -75,6 +75,7 @@ struct ogg_stream {
     int incomplete; ///< whether we're expecting a continuation in the next page
     int page_end;   ///< current packet is the last one completed in the page
     int keyframe_seek;
+    int got_start;
     void *private;
 };
 
